resources/js/Components/InfoBlock.vue

```diff
@@ -1,27 +1,47 @@
 <script setup lang="ts">
-import { MapPinIcon as DefaultIcon } from '@heroicons/vue/24/solid'
-
-defineProps<{
-  icon?: any
-  title: string
-  line1?: string
-  line2?: string
-  line3?: string
-}>()
+import { UserIcon as DefaultIcon } from '@heroicons/vue/24/outline'
+const props = withDefaults(
+  defineProps<{
+    icon?: object | null
+    imageUrl?: string | null
+    title?: string | null
+    info1?: string | null
+    info2?: string | null
+    note?: string | null
+  }>(),
+  {
+    icon: DefaultIcon,
+    imageUrl: undefined,
+    title: '',
+    info1: '',
+    info2: '',
+    note: '',
+  },
+)
 </script>
 
 <template>
-  <div class="flex gap-x-6 items-center">
+  <div class="flex sm:gap-x-6 gap-x-3 items-center">
     <div
-      class="flex bg-brand/10 size-20 rounded-2xl items-center justify-center aspect-square"
+      class="flex shrink-0 bg-brand/10 sm:size-20 size-10 rounded-2xl items-center justify-center aspect-square overflow-hidden"
     >
-      <component :is="icon || DefaultIcon" class="size-10 text-brand" />
+      <img
+        v-if="props.imageUrl"
+        :src="props.imageUrl"
+        alt="Obrazek"
+        class="object-cover size-full"
+      >
+      <component
+        :is="props.icon || DefaultIcon"
```

Comment:
I think you meant to use `??` instead of  `||`. Do you know differences ?

```suggestion
        :is="props.icon ?? DefaultIcon"
```